packages/@uppy/vue/types/components.d.ts

```diff
@@ -0,0 +1,44 @@
+import Vue from 'vue'
+import type { Uppy, Plugin } from '@uppy/core'
+
+declare module '@uppy/vue' {
```

Comment:
Is this file still necessary? It looks like these exports are now declared by individual typescript files instead and reexported from `index.d.ts`?

Reply:
No it isn't necessary anymore, I'll delete it